l10n_be_cooperator/report/tax_shelter_report_templates.xml

```diff
@@ -12,32 +12,20 @@ SPDX-License-Identifier: AGPL-3.0-or-later
                 <td class="text-start">
                     <span t-field="line.type" />
                 </td>
-                <td class="text-end">
-                    <span
-                        t-field="line.capital_before_sub"
-                        t-options='{"widget": "monetary", "display_currency": o.company_id.currency_id}'
-                    />
-                </td>
-                <td class="text-end">
-                    <span
-                        t-field="line.capital_after_sub"
-                        t-options='{"widget": "monetary", "display_currency": o.company_id.currency_id}'
-                    />
-                </td>
-                <td class="text-center">
+                <td class="text-start">
                     <span t-if="line.tax_shelter">Yes</span>
                     <span t-else="">No</span>
                 </td>
                 <td class="text-start">
                     <span t-field="line.transaction_date" />
                 </td>
-                <td class="text-end">
+                <td class="text-start">
```

Comment:
these were aligned at the end to ensure decimals line up in case there are multiple lines. but since there is no total, i think it’s ok like this, and it looks better, indeed.